webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
This seems unnecessary since we're producing our own html. What's the purpose of this?

Reply:
Removed HtmlWebpackPlugin since we do not need it to create HTML.